src/planner/vectorfield/VectorFieldPlannerExceptions.cpp

```diff
@@ -0,0 +1,31 @@
+#include "VectorFieldPlannerExceptions.hpp"
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+/*
+ * VectorFieldTerminated
+ */
```

Comment:
Nit: Please remove these redundant comments and add the function definition splitter.